flashinfer/grouped_mm/kernels/cutedsl/sm120_moe/moe_gemm_mxfp8_mxfp4.py

```
@@ -0,0 +1,664 @@
+                    store_output.store_predicated(
+                        st_atom, thr_st, tDrD, gD_tile, (bm, bn),
+                        tile.m_offset + tile.m_block * bm, tile.n_block * bn, tile.m_boundary, N)
+                tile = cons.get_next_tile(sWork, sfull, sempty)
+
+
+def _stream():
+    return cuda.CUstream(torch.cuda.current_stream().cuda_stream)
+
+
+def make_args(a_q, b_q, a_scale, b_scale, out, m_indptr):
+    sf = lambda t: from_dlpack(t.contiguous(), assumed_align=16).mark_layout_dynamic()
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `sf` as a `def`

(E731)